code/lighting/lighting_profiles.cpp

```diff
@@ -96,6 +257,38 @@ void lighting_profile::parse_default_section()
 		parsed |= parse_optional_float_into("$PPC Shoulder Length:",&default_profile.ppc_values.shoulder_length);
 		parsed |= parse_optional_float_into("$PPC Shoulder Strength:",&default_profile.ppc_values.shoulder_strength);
 		parsed |= parse_optional_float_into("$PPC Shoulder Angle:",&default_profile.ppc_values.shoulder_angle);
+		parsed |= parse_optional_float_into("$Exposure:",&default_profile.exposure);
+
+		parsed |= lighting_profile_value::parse(filename,"$Missile light brightness:","Default profile",
```

Comment:
I would suggest setting a `const char *` variable to "Default profile" and then using that for each line.